TODO.org

```diff
@@ -72,3 +72,4 @@
 *** TODO mount boards
 *** TODO mount connectors
 *** TODO label and close it up
+** TODO design and assemble the equipment cabinet
```

Comment:
whoops, i think this was pulled in when you merged my jdb-bgcolor branch. sorry about that.